forecasting/src/autogluon/forecasting/learner.py

```diff
@@ -0,0 +1,139 @@
+    """ForecastingLearner encompasses a full time series learning problem for a
+    training run, and keeps track of datasets, features, random seeds, and the
+    trainer object.
+    """
+
+    def __init__(
+        self,
+        path_context: str,
+        random_state: int = 0,
+        trainer_type: Type[AbstractForecastingTrainer] = AutoForecastingTrainer,
+        eval_metric: Optional[str] = None,
+    ):
+        super().__init__(path_context=path_context, random_state=random_state)
+        self.eval_metric: str = check_get_evaluation_metric(eval_metric)
+        self.trainer_type = trainer_type
+        logger.info(f"Learner random seed set to {random_state}")
+
+    def load_trainer(self) -> AbstractForecastingTrainer:
+        """Return the trainer object corresponding to the learner."""
+        return super().load_trainer()  # noqa
+
+    def fit(
+        self,
+        train_data: TimeSeriesDataFrame,
+        prediction_length: int,
+        val_data: TimeSeriesDataFrame = None,
+        hyperparameters: Union[str, Dict] = None,
+        hyperparameter_tune: bool = False,
+        **kwargs,
+    ) -> None:
+        return self._fit(
+            train_data=train_data,
+            prediction_length=prediction_length,
+            val_data=val_data,
+            hyperparameters=hyperparameters,
+            hyperparameter_tune=hyperparameter_tune,
+            **kwargs,
+        )
+
+    def _fit(
+        self,
+        train_data: TimeSeriesDataFrame,
+        prediction_length: int,
+        val_data: Optional[TimeSeriesDataFrame] = None,
+        hyperparameters: Union[str, Dict] = None,
+        hyperparameter_tune: bool = False,
+        scheduler_options: Tuple[Type, Dict[str, Any]] = None,
+        time_limit: Optional[int] = None,
+        **kwargs,
+    ) -> None:
+        self._time_limit = time_limit
+        time_start = time.time()
+
+        logger.debug(
+            "Beginning AutoGluon training with ForecastingLearner "
+            + (f"Time limit = {time_limit}" if time_limit else "")
+        )
+        logger.info(f"AutoGluon will save models to {self.path}")
+
+        self.trainer = self.trainer_type(
+            path=self.model_context,
+            prediction_length=prediction_length,
+            eval_metric=self.eval_metric,
+            scheduler_options=scheduler_options,
+            **kwargs,
+        )
+        self.trainer_path = self.trainer.path
+        self.save()
+
+        self.trainer.fit(
+            train_data=train_data,
+            val_data=val_data,
+            hyperparameter_tune=hyperparameter_tune,
+            hyperparameters=hyperparameters,
+            time_limit=time_limit,
+        )
+        self.save_trainer(trainer=self.trainer)
+
+        self._time_fit_training = time.time() - time_start
+        logger.info(
+            f"AutoGluon ForecastingLearner complete, total runtime = {round(self._time_fit_training, 2)}s",
+        )
```

Comment:
logger.log(20, ...) is what is used for Tabular for these kinds of logs, otherwise .info won't show up by default I think.

Reply:
I believe `logger.info` is pretty much an alias for `logger.log(20, ...`. See [here](https://github.com/python/cpython/blob/f545fc955aeb701ae4e73b07ff2283f823d857b8/Lib/logging/__init__.py#L1504) for example. Do you have any example where they are not equivalent?

If they are equivalent I would prefer `logger.info` as it's more explicit.

Reply:
oh you are correct, for some reason I though info was 15, but it is 20. Feel free to keep it as is then